targets/TARGET_Maxim/TARGET_MAX32625/rtc_api.c

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (C) 2016 Maxim Integrated Products, Inc., All Rights Reserved.
+ * Copyright (C) 2016,2018 Maxim Integrated Products, Inc., All Rights Reserved.
```

Comment:
Interesting. Is this a valid license header? I've never seen one with the years split like this.